zeebe/protocol/src/main/resources/cluster-management-protocol.xml

```diff
@@ -57,6 +57,7 @@
     <field name="partitionId" id="1" type="uint16"/>
     <field name="type" id="2" type="BackupRequestType"/>
     <field name="backupId" id="3" type="int64"/>
+    <data name="pattern" id="4" type="varDataEncoding"/>
```

Comment:
❓ Should some of them be set as optional? 🤔

Reply:
`varDataEncoding` can't be optional, sadly.